doc/man1/openssl-x509.pod.in

```diff
@@ -165,14 +165,17 @@ Names and values of these options are algorithm-specific.
 
 =item B<-key> I<filename>|I<uri>
 
-This option provides the private key for signing a new certificate or
-certificate request.
+This option usually provides the key pair for self-signing a new certificate
```

Comment:
"usually" is a wishy-washy word. Could this be re-phrased?

Reply:
Thanks  @tmshort for your comment and prospective approval.
I've improved the wording, no more using "usually".

Can you and @t8m please re-approve.